internal/rollup/rollup.config.spec.js

```diff
@@ -72,7 +72,7 @@ describe('rollup config', () => {
     expect(doResolve(`other${sep}thing`))
         .toEqual(`${baseDir}/bazel-bin/path/to/a.esm5/external/other_wksp/path/to/other_lib/thing`);
     expect(doResolve(`@bar${sep}baz${sep}foo`))
-      .toEqual(`${baseDir}/bazel-bin/path/to/a.esm5/path/to/bar/baz_lib/foo`);        
+        .toEqual(`${baseDir}/bazel-bin/path/to/a.esm5/path/to/bar/baz_lib/foo`);
```

Comment:
why formatting changes? could you generally go through the PR and remove non-load-bearing deltas so it's easier to review?

Reply:
Looking at this file though the formatting fix seems correct:
https://github.com/bazelbuild/rules_nodejs/blob/master/internal/rollup/rollup.config.spec.js#L75
This one line has indentation out of order and trailing whitespaces which does seem incorrect to me.

Either way I will remove it.